ckan_cloud_operator/providers/ckan/deployment/drone/manager.py

```diff
@@ -0,0 +1,92 @@
+import requests
+from urllib.parse import urljoin
+
+from ...env import manager as env_manager
+
+
+class Drone(object):
+
+    def initialize(self, force_prompt=False):
+        self.conf = env_manager._read_yaml().get('cicd', {})
+        self.server_url = self.conf.get('serverUrl')
+        self.api_token = self.conf.get('token')
+        self.org = self.conf.get('organization')
+        self.repo = self.conf.get('repo')
+        if force_prompt or not self.conf:
+            self.server_url = input('Please enter Drone Server URL: ')
+            self.api_token = input(f'Please enter Drone API Tocken. See {self. server_url}acount: ')
+            self.org = input('Please enter Github organization name: ')
+            self.repo = input('Please enter Github repository name: ')
+            self.conf['cicd'] = {
+                 'serverUrl': self.server_url,
+                 'token': self.api_token,
+                 'organization': self.org,
+                 'repo': self.repo
+            }
+            env_manager._write_yaml(self.conf)
+        self.api_url = urljoin(self.server_url, '/'.join(['api/repos', f'{self.org}/{self.repo}', 'builds/']))
+        self.header = {'Authorization': f'Bearer {self.api_token}'}
+        print('CCO is configured for Drone ')
+
+
+    def builds_list(self):
+        resp = requests.get(self.api_url, headers=self.header)
+        if not _check_for_200(resp):
+            return []
```

Comment:
```suggestion
            return {}
```
Better return an empty json if the expected return value is a json object.

Reply:
response is a list of build info. So this needs to be list